core/src/filed/dir_cmd.cc

```diff
@@ -2001,7 +2004,10 @@ static bool BackupCmd(JobControlRecord* jcr)
 
   // Send Files to Storage daemon
   Dmsg1(110, "begin blast ff=%p\n", (FindFilesPacket*)jcr->impl->ff);
-  if (!BlastDataToStorageDaemon(jcr, nullptr, cipher)) {
+
+
+  if (!BlastDataToStorageDaemon(jcr, nullptr, cipher,
+                                client->max_network_buffer_size)) {
```

Comment:
The bufsize setting is not equivalent to what was moved out of the function.

Reply:
i am not sure what this means